src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -472,8 +472,7 @@ public String getDlnaContentFeatures(RendererConfiguration mediaRenderer) {
 		String dlnaOrgPnFlags = getDlnaOrgPnFlags(mediaRenderer, localizationValue);
 		return (dlnaOrgPnFlags != null ? (dlnaOrgPnFlags + ";") : "") 
 //				+ getDlnaOrgOpFlags(mediaRenderer) 
-				// Time seek is tricky - as of now, don't support it
-				+ "DLNA.ORG_OP=01;DLNA.ORG_CI=0;DLNA.ORG_FLAGS=01700000000000000000000000000000";
+				+ "DLNA.ORG_OP=11;DLNA.ORG_CI=1;DLNA.ORG_FLAGS=01700000000000000000000000000000";
 	}
```

Comment:
@atamariya That's a nice catch :+1: 
I didn't readed this part of the standard yet ;-)

I will test it later on transcoded and not transcoded videos.